### ISSUE TITLE
Nothing works

### ISSUE THREAD

**Issue by kstenerud** [microsoft/vscode-go#2792]
Steps to Reproduce:

1. Install Visual Studio (Ubuntu 18.04)
2. Open the extensions and install the go extension
3. Install/update tools as per https://github.com/Microsoft/vscode-go/wiki/Go-with-VS-Code-FAQ-and-Troubleshooting
4. Open a go project directory

When I do this, none of the functions works. I can't see syntax errors. Intellisense doesn't work. Code completion doesn't work. Go to definition doesn't work. If there are other features, I don't know about them because nothing other than the linter seems to do anything.

Basically, VS Code is a glorified text editor, and I do all my dev in the command line :/

**Comment by roflomglol** [comment]
I'm not sure if I have related issue because at least something works for me(linters work only in _test files maybe because output is different), but I had to restore my VSCode setup from backup and it worked as before until I received an update of Go extention.

Formatting works for me, as do Go to definitions, but build on save workspace doesn't notice any syntax errors. Not in current package nor in dependent packages.

It seems there are maybe some issues with build on save workspace, because if I switch to package - it works. If I open a console it seems build hangs or just taking enormous amount of time. Problems page is clean too even though I have some output for example from go vet that points to some issues with code.
